product_customer_code_sale/models/sale.py

```diff
@@ -16,6 +16,7 @@ def _compute_get_product_customer_code(self):
             product = line.product_id
             code_id = product_supplierinfo_obj.search([
                 ('product_tmpl_id', '=', product.product_tmpl_id.id),
+                ('type', '=', 'customer'),
```

Comment:
This is good. However I miss also the filter for the customer. `('name', '=', line.order_id.partner_id),` would do the thing I think